examples/sideinput/simple_sideinput/README.md

```diff
@@ -43,9 +43,28 @@ Side input spec:
 ```yaml
```

Comment:
Keep the original as is, and add ab example section

Reply:
Or you mean removing the image values etc and just keeping the structure?